src/main.ts

```diff
@@ -0,0 +1,8 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  const app = await NestFactory.create(AppModule);
+  await app.listen(3000);
```

Comment:
Move port value to NestJS config file.
https://docs.nestjs.com/techniques/configuration#using-the-configservice